main/src/CoffeeMachineSimulator.java

```diff
@@ -0,0 +1,200 @@
+import java.util.Scanner;
+
+public class CoffeeMachineSimulator {
+
+    //declaring our class variables
+    int water = 400;
+    int milk = 540;
+    int coffeeBeans = 120;
+    int disCups = 9;
+    int money = 550;
+    String buy = "buy";
+    String fill = "fill";
+    String take = "take";
+    String exit = "exit";
+    String remaining = "remaining";
+    String back = "back";
+    int mode = 1;
+
+
+    // method myMethod check coffee machine mode, and call appropriate method
+    public void checkMethod(String inputString) {
```

Comment:
@Edvard777 please change method name it is not descriptive , and the variable name as well . Also you can try to use java doc comments , write `/**` press enter comment will be generated . And in the /** **/ write your comment

Reply:
done